### PR TITLE
Create anonymizationMethodUsed.md

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
Hi @kestewart @bennetkl Please review

### CODE REVIEW

**Review by kestewart — CHANGES_REQUESTED** [review]
Fix typo "anonymyze" in summary. 
In metadata name shouldn't be sensors.

File: model/Dataset/Properties/anonymizationMethodUsed.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Description
+
+The field describes the anonymization methods used. If multiple anonymization methods are used, each anonymization method is specified in a new line. 
+
+## Metadata
+
+- name: sensors
+- Nature: DataProperty
+- Range: xsd: string
```
Comment (armintaenzertng) [inline model/Dataset/Properties/anonymizationMethodUsed.md:17]:
```suggestion
- Range: xsd:string
```

File: model/Dataset/Properties/anonymizationMethodUsed.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# anonymizationMethodUsed
+
+## Summary
+
+The field describes the methods used to anonymyze the dataset (of fields in the dataset).
+
+## Description
+
+The field describes the anonymization methods used. If multiple anonymization methods are used, each anonymization method is specified in a new line.
```
Comment (armintaenzertng) [inline model/Dataset/Properties/anonymizationMethodUsed.md:11]: Same comment as [here](https://github.com/spdx/spdx-3-model/pull/163#discussion_r1156985033).
That is, `anonymizationMethodUsed` has arbitrary cardinality when used in the [Dataset class](https://github.com/spdx/spdx-3-model/blob/dataset-profile/model/Dataset/Classes/Dataset.md), so we should not discuss multiple values in this property.

**Review by bennetkl — COMMENTED** [review]
Typically with AI applications, data anonymization is done by creating a mirror image of the dataset and then  implementing alteration strategies, such as character shuffling, encryption, term, or character substitution.     This field is meant to capture all the strategies used to 'change' the dataset

**Comment by goneall** [comment]
Superseded by PR #185 which includes feedback form this PR

**Pull request closed without merging**